<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>491</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Choices :</string>
    </property>
    <widget class="QLineEdit" name="choices1">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>20</y>
       <width>471</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>241</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Question 2 :</string>
    </property>
    <widget class="QTextEdit" name="question2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>221</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>241</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Question 3 :</string>
    </property>
    <widget class="QTextEdit" name="question3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>221</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>280</y>
      <width>491</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Choices :</string>
    </property>
    <widget class="QLineEdit" name="choices2">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>20</y>
       <width>471</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Difficulty :</string>
    </property>
    <widget class="QLineEdit" name="difficultyLine">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>241</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Question 1 :</string>
    </property>
    <widget class="QTextEdit" name="question1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>221</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Category :</string>
    </property>
    <widget class="QLineEdit" name="categoryLine">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>21</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>450</y>
      <width>491</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Choices :</string>
    </property>
    <widget class="QLineEdit" name="choices3">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>20</y>
       <width>471</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
